CHANGE_LOG

```diff
@@ -1,3 +1,54 @@
+v0.44.0rc1 (Nov 8, 2024)
+------------------------
+
+Highlights of this release are:
+
+- Official support for Python 3.13.
+- Dropped official support for Python 3.9, the minimum supported Python version is 3.10.
+- LLVM 15 is now the default LLVM.
+- Added support for LLVM's new PassManager.
+- Support for LLVM based process triple partitioning.
```

Comment:
```suggestion
- Support for LLVM based target triple partitioning.
```
it's `target` not `process`, the process can provide its associated target triple but this functionality consumes a target triple string.